src/searchForm/autoComplete/overlay/AutoCompleteOverlay.tsx

```diff
@@ -14,24 +14,38 @@ export type AutoCompleteOverlayProps = Omit<
   readonly renderAutocompleteComponent: (
     props: Omit<AutoCompleteProps, 'searchForItems'>,
   ) => JSX.Element
+  readonly closeOnBlur: () => void
```

Comment:
Just saw that the `readonly` (and `Readonly<T>`) are not required for React components' props and state because they already apply it.

[Explained here](https://raulfdm.github.io/react-typescript-cheatsheet/docs/basic/getting-started/class_components) under the question "No need for readonly".

Reply:
Good to know, thanks